--- a/asyncssh/crypto/chacha.py
+++ b/asyncssh/crypto/chacha.py
@@ -26,12 +26,13 @@
 from cryptography.hazmat.backends.openssl import backend
 from cryptography.hazmat.primitives.ciphers import Cipher
 from cryptography.hazmat.primitives.ciphers.algorithms import ChaCha20
-from cryptography.hazmat.primitives.poly1305 import Poly1305
 
 from .cipher import register_cipher
 
 
-if backend.poly1305_supported():
+if hasattr(backend, 'poly1305_supported') and backend.poly1305_supported():
+    from cryptography.hazmat.primitives.poly1305 import Poly1305
+
     _CTR_0 = (0).to_bytes(8, 'little')
     _CTR_1 = (1).to_bytes(8, 'little')
 
